ansible/server-state-playbooks/www/tests/check_redirects.py

```diff
@@ -10,15 +10,12 @@
 import pytest
 import requests
 
-
 HOST = os.getenv('HOST', 'https://ome-www.openmicroscopy.org')
```

Comment:
You may wish to start testing the `www.openmicroscopy.org` address now the DNS points to this server. To rule out any issues with configurations on the server being different based on the hostname.